docs/spec/interfaces.md

```diff
@@ -0,0 +1,80 @@
+# Interface Contracts
+
+<!-- This image is sourced from https://drive.google.com/open?id=1Mgn0Oz85M9W1Pwfnw3u0OhZmIIxSxcM3lZUEZyPCoB0 -->
+
+![Interface Contracts Overview](images/interface-contracts-overview.svg)
+
+---
+
+## Subscribable
```

Comment:
I don't think Subscribable needs to be a public interface now that the only valid target of a Subscription is a Channel.

Reply:
for now.

Reply:
This is already in the spec where subscribable is used, the limitation is channel only.